Detectors/MUON/MCH/Clustering/src/ClusterOriginal.h

```diff
@@ -17,6 +17,7 @@
 #ifndef O2_MCH_CLUSTERORIGINAL_H_
 #define O2_MCH_CLUSTERORIGINAL_H_
 
+#include "PadOriginal.h"
```

Comment:
Maybe a detail, but if you add the include, you can remove the forward declaration

Reply:
Good point. Fixed.